dune-project

```diff
@@ -56,7 +56,7 @@ possible and does not make any assumptions about IO.
   ordering
   dune-build-info
   spawn
-  cmarkit
+  (cmarkit (= 0.1.0))
```

Comment:
cmarkit 0.2.0 is now on opam-repository, so we can depend on >= 0.2.0 and get rid of the temporary fix

Reply:
https://github.com/ocaml/opam-repository/pull/23781